developer-docs-site/docs/tutorials/first-fungible-asset.md

```diff
@@ -0,0 +1,247 @@
+---
+title: "Your First Fungible Asset"
+slug: "your-first-fungible-asset"
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Your First Fungible Asset
+
+This tutorial introduces how you can compile, deploy, and mint your own fungible asset (FA), named [Moon Coin](https://github.com/aptos-labs/aptos-core/tree/main/aptos-move/move-examples/fungible_asset/moon_coin).
+Make sure you have understood FA before moving on to the tutorial. If not, it is highly recommended to read it first.
+
+* [Fungible Asset](../standards/fungible-asset.md)
+
+## Step 1: Pick an SDK
```

Comment:
I wonder if we can point to a common section for setting up sdk and cli and cloning aptos-core as all tutorials need those same steps. Cc @saharct